### PR TITLE
Data analysis

### PR DESCRIPTION
**Pull request by adityavarshney**
Added ipynb notebooks containing EDA and Twitch user sampling methods.
Added csv containing 2000 sampled channels (for initial model training)
Created model API in model.py with first version of clustering algorithm and output clusters.

### CODE REVIEW

**Comment by scottnuma** [comment]
Running the application now display 10 top streamers from the group.

Possible ways to make more related - add randomized selection of streamers from group.
Errors on display of `Riot Games` and some other group 0 top 10 streamers.

**Comment by scottnuma** [comment]
`create_model` endpoint creates a full fledged and usable data frame for pulling formation out of.

**Comment by kcempron** [comment]
@scottnuma @adityavarshney what's the status of this PR? is it ready for merge?

File: flask-web/templates/about.html [hunk 1]
```diff
@@ -19,8 +20,8 @@ <h2>Our Story</h2>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vehicula libero auctor dolor vulputate varius. Nunc ac nisl non felis finibus ultricies. Donec at porttitor lorem. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Proin volutpat augue ex. In hac habitasse platea dictumst. Suspendisse at ante ligula. Nulla facilisis velit egestas massa imperdiet, id posuere turpis feugiat.
             </p>
             <br>
-            <p>  
-               Etiam at consectetur sapien, vel laoreet diam. Aliquam erat volutpat. Quisque fermentum purus ac justo fermentum vulputate eu id libero. Donec quam ipsum, malesuada nec interdum in, commodo vel nisl. Aliquam erat volutpat. Donec suscipit molestie urna, at pharetra tortor interdum eget. Ut lobortis viverra ex, quis faucibus sapien iaculis in. Suspendisse laoreet ornare justo sit amet fermentum. Pellentesque non nulla id libero pretium bibendum vel nec dolor. 
+            <p>
```
Comment (kcempron) [inline flask-web/templates/about.html]: Can we update this about text with actual text? Im sure there is something we can pull from the poster info doc

**Pull request merged**